code/game/gamemodes/wizard/artefact.dm

```diff
@@ -880,3 +882,20 @@ var/global/list/multiverse = list()
 	heal_brute = 25
 	heal_burn = 25
 	heal_oxy = 25
+
+/mob/living/proc/skeleton_reset()
+	set name = "Offer skeleton to ghosts"
+	set category = "Skeleton"
+	set desc = "Give up your skeleton to ghosts so you can observe the round."
+	for(var/mob/living/carbon/human/G in GLOB.mob_list)
```

Comment:
Can you check if this works fine without this `for` loop? It seems kind of pointless given you can just refer to `src` directly, instead of having to find `src` in the mob list first.

Reply:
thanks for the reminder, works fine without the loop